cmd/aws/create.go

```diff
@@ -41,7 +46,14 @@ func createAws(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("invalid catalog apps: %w", err)
 	}
 
-	err = ValidateProvidedFlags(cliFlags.GitProvider)
+	ctx := context.Background()
```

Comment:
```suggestion
	ctx := cmd.Context()
```

https://pkg.go.dev/github.com/spf13/cobra#Command.Context